pkg/kubelet/volumes.go

```diff
@@ -106,11 +99,13 @@ func (vh *volumeHost) GetWriter() io.Writer {
 	return vh.kubelet.writer
 }
 
-// Returns the hostname of the host kubelet is running on
```

Comment:
@pmorie  previous comment said those are out of scope and I originally godoc'd all of them but he said they should be removed for now - although I think that Paul comment was in slack an not an official comment in this PR - that one line you reference above existed before I did all the other Get* functions, so I just removed it as well to be consistent...will put back in if you think I should @swagiaal @pmorie